microsoft/testsuites/dpdk/dpdkperf.py

```diff
@@ -164,6 +250,25 @@ def perf_dpdk_netvsc_pmd_multi_queue(
 
         self._run_dpdk_perf_test("netvsc", environment, log, variables, use_queues=True)
 
+    @TestCaseMetadata(
+        description="""
+        DPDK Performance: direct use of VF, maximum core count, maximum tx/rx queues,
+        Run on a huge machine
+        """,
+        priority=3,
+        requirement=simple_requirement(
+            min_count=2, network_interface=Sriov(), min_nic_count=2, min_core_count=48
+        ),
+    )
+    def perf_dpdk_netvsc_pmd_multi_queue_huge_vm(
+        self,
+        environment: Environment,
+        log: Logger,
+        variables: Dict[str, Any],
+    ) -> None:
+
+        self._run_dpdk_perf_test("netvsc", environment, log, variables, use_queues=True)
```

Comment:
Why some cases check the core count, but some of them not?

Reply:
If it's needed, can this check moves into `_run_dpdk_perf_test`?

Reply:
There's a default number of cores for the tests that line up with the minimum required cores for those tests. For the test that use larger amounts of cores the 'run concurrent' function takes in a single argument and assumes the VMs are the same size. I wanted to double check we aren't passing a core argument that DPDK can't fulfill on one of the VMs. Honestly now that we have the minimum core count argument on the tests themselves this check is probably redundant.

Reply:
For the statement below, it looks like "dpdk expects" all core should be the same. So I don't understand why not check the core count in `_run_dpdk_perf_test`, but only check in some of test cases.


Nodes contain different core counts, DPDK Suite expects sender and receiver to have same core count.

Reply:
scrooched it to the run function